doc/how_to_build_win.md

```diff
@@ -119,6 +121,12 @@ If a previous binary of OpenToonz was already installed, this step and the follo
 ### Running
 `OpenToonz_1.2.exe` can now be run.  Congratulations!
 
+## Debugging
+1. Start OpenToonz from `$opentoonz/toonz/build/Debug`.
+  - You must follow the same instructions to deploy and run OpenToonz first. All files must be present in the `Debug` folder as well.
+2. In Visual Studio, in the command bar, navigate to `Debug -> Attach to Process...` (Ctrl+Alt+P). Find `OpenToonz_1.2.exe` and attach to it.
+  - You might have to restart Visual Studio to elevate its privileges first, then redo step #2.
```

Comment:
This whole debugging section shouldn't  be necessary.  If you have debug selected from the drop down and hit f5, it will open OpenToonz if all the files are in the right place.

Reply:
It says access to ALL_BUILD is denied. @manongjohn gave me that trick.

Reply:
I have problems with Debug. Doesn't want to work for me.  Not sure why. I have to use RelWithDebugInfo and attach to process. Problem with my approach is I can't debug as OT starts up if i need to.

Reply:
I searched on the Internet, and it seems my problem is known, but has (maybe) only been fixed in VS2017. I tried resetting file permissions, and using Managed Compatibility mode, but it does not work. I can still attach to a Debug build, though.

Reply:
@jpturcotte

> It says access to ALL_BUILD is denied. 

 Have you already tried the following?
- In Solution Explorer, select the `OpenToonz_1.2` project within `OpenToonz` solution.
- On the `Project` menu, choose `Set as StartUp Project` .

Reply:
@shun-iwasawa It works! Thank you very much! I was at a loss as to what to do. I will update the instructions. Can I ask you how to clean up my messy commit history, please?

Reply:
I think it will work. I recommend you to keep the branch's backup ( `git branch how-to-build-win_bkup how-to-build-win` ) in advance.

- Register upstream (only if you hadn't done in the past)

`> git remote add upstream https://github.com/opentoonz/opentoonz`

- Move to your local master branch

`> git checkout master`

- Synchronize your local master branch with upstream

`> git pull upstream master`

- Move to the topic branch

`> git checkout how-to-build-win`

- Rebase the topic branch to the current master

`> git rebase master`

- Push the topic branch again. `-f` (force update) option is needed because you rebased the branch.

`> git push -f origin how-to-build-win`